cohort-cli/src/test/resources/cql/zip/FHIRHelpers.xml

```diff
@@ -1,4 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--  ************************************** -->
```

Comment:
Maybe worth talking about at standup - we didn't author these CQL files, we copied them.  Putting an IBM copyright on them feels wrong.

Reply:
Didn't realize they were non-IBM authored, I will remove.